lab_1_classify_by_unigrams/main.py

```diff
@@ -63,6 +129,23 @@ def detect_language(
     :param profile_2: a dictionary of a known profile
     :return: a language
     """
+    if isinstance(unknown_profile, dict) == False:
+        return None
+    if isinstance(profile_1, dict) == False or isinstance(profile_2, dict) == False:
+        return None
+    mse_1 = compare_profiles(unknown_profile, profile_1)
+    mse_2 = compare_profiles(unknown_profile, profile_2)
+    if mse_1 == mse_2:
+        names = []
```

Comment:
consider directly creating a list of names via `[profile_1['name'], profile_2['name']]`

Reply:
fixed